Jbarget/oxymore
# So/phase 1 finishing touches

### What changes have you made?
Phase 1 of the site needed a few finishing touches before going live which I decided to tackle on one PR:

- imported Favorit and applied it globally - the fallback is still `sans-serif`
- added some extra hover styles on the Home, Menu and Buy buttons for a bit more of an interactive look and feel
- switched around the images on the Advertising page and also did a refactor to tighten up the code a bit
- some backgrounds have been updated from marble to black from the `Background` component 
- a gradient colour has been set as a background on the nav menu to create a visual contrast on click
- fixed an issue where the home button on the nav menu was not routing back to the home page 
- increased the font size on the About page to be the same size as its `Header` 
- improved the symmetricity on the manifesto paragraphs and merged the two translations to create a bit of a snake-effect
- made a check on the SangBleu font file and realised there was a typo in the `font-family` (it should be SangBleu OG Serif not SangBleu) this has now been fixed in `GlobalStyle` 
- tidied up the spacing on the About page

 ###  #020403 is the new black 
- previously we were using standard colours like black - this is a bit heavy on the eye. So I've applied a softer variation of black in the `colorCodes` object of the `theme`


### Which issue(s) does this PR fix?

Fixes #297 
Fixes #303 
Fixes #295 
Fixes #295 
Fixes #293 
Fixes #304 
Fixes #306 

### Screenshots (if there are design changes)
Gradient background:

<img width="280" alt="Screenshot 2020-12-29 at 18 24 45" src="https://user-images.githubusercontent.com/53219789/103305559-579a0100-4a03-11eb-8207-393474d9ff4d.png">

Favorit example:

<img width="280" alt="Screenshot 2020-12-29 at 18 25 02" src="https://user-images.githubusercontent.com/53219789/103305564-5963c480-4a03-11eb-8a48-0a2b5b3ff537.png">


### How to test
Move through the site and look out for: 
- any instances where the font reverts to the fallback - Safari, Firefox..?
- the console is picking up the correct hex code `#020403` 
- the new hover styles on the buttons are all working
- I also picked up a small issue on the contact page where the email address was not opening out into an email template on click. I had to update my default email app settings to get this to work. Check to see if you have the same issue.
